tests/angular_devkit/build_angular/test-differential-loading/tsconfig.app.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I am not quite sure why we need to add all these tsconfig.json files.

The large spec, should cover an existing application that we have a large spec.

Reply:
Below we have an example of a tsconfig cycle, this is probably not covered by our existing app?

Maybe `readTsconfig` covers this?

Reply:
Yes, this was the idea. As I was not aware of `readTsconfig`, I wrote this by my self. And hence, I wanted to test the logic with a tsconfig pointing to ES5, a tsconfig pointing to ES2015, a tsconfig w/o a target pointing to another tsconfig with a extends cycle. 

Now, as I'm using `readTsconfig` this does not make sense anymore as we can assume that this function which is part of typescript does its job.